ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ActiveContracts.scala

```diff
@@ -193,7 +144,8 @@ class ActiveContractsManager[ACS](initialState: => ACS)(implicit ACS: ACS => Act
                     })
                   } else {
                     acc
-                  })
+                  }),
+                  parties = parties.union(nodeParties)
```

Comment:
Why is there no `AddTransactionState` in this branch?

Reply:
The above line is part of the `ats.copy` call, which copies the current `AddTransactionState`.